Add syndicated version of Support Solidus Article from Karma Creative

MFRWDesign opened the pull request:
This PR adds a new post for the syndicated version of the Support Solidus Article.

@aldesantis @kennyadsl I know we've been talking about some upcoming changes; please let me know if you feel this and/or the original article needs any update (now or in the near future to reflect our current policies).

aldesantis commented:
I think it's fine as is — if needed, we can revise it in the future or, even better, post an update.

aldesantis commented on source/blog/2020-06-24-support-solidus.html.markdown:
```diff
@@ -0,0 +1,101 @@
+In summary, the Support Solidus initiative was meant, first and foremost, as a *call to arms*. With your code, through becoming an Ambassador, or by donating your time, we want to remind everyone that, together, it can be done.
+
+---
+_To continue reading this article, please visit the original on the [Karma Creative](https://karmacreative.io) website: [Full Text of original Support Solidus Article](https://karmacreative.io/support-solidus)._
```
I'm a bit concerned by the UX implications of asking people to visit another site to continue reading, although I'm on expert here and don't really have a strong opinion. @kennyadsl what are your thoughts?

kennyadsl replied:
I don't have any specific concerns around continue reading on an external website but for me, the reasoning behind this is just not clear. If it's for SEO reasons, we have the canonical tag and should not impact.

aldesantis commented:
@MFRWDesign can you update the PR (including the date of the post) and then we can merge?

MFRWDesign commented:
@aldesantis Sorry for the massive delay here - just got all tied up with client work and these changes kept getting pushed down in the stack. This should again be ready for review.

I changed the CTA at the bottom of the syndicated version of the article to point to Open Collective, and I updated the publication date.

(FYI My rationale for not including the whole article was the it was extremely long and didn't fit the flow of the rest of the Solidus blog, not to mention it would've required adding a bunch more pictures, adapting styles, and creating a ToC element. -- We'll circle back around on this in the future and can come back to make an update here when we have time.)

kennyadsl reviewed (approved):
Works for me, thanks @MFRWDesign !
